QRCoder/PayloadGenerator/Girocode.cs

```diff
@@ -49,22 +54,26 @@ public Girocode(string iban, string bic, string name, decimal amount, string rem
                 if (amount.ToString().Replace(",", ".").Contains(".") && amount.ToString().Replace(",",".").Split('.')[1].TrimEnd('0').Length > 2)
                     throw new GirocodeException("Amount must have less than 3 digits after decimal point.");
                 if (amount < 0.01m || amount > 999999999.99m)
-                    throw new GirocodeException("Amount has to at least 0.01 and must be smaller or equal to 999999999.99.");
+                    throw new GirocodeException("Amount has to be at least 0.01 and must be smaller or equal to 999999999.99.");
```

Comment:
Can you please fix the test cases that do assertions on this exception? (See Github action runner. Test cases seem to be broken.)

Reply:
Certainly